Makefile

```diff
@@ -27,6 +27,16 @@ quotelint:
 		exit 1;                                                       \
 	fi >&2
 
+check-copyright:
+	@found="";                                                                              \
+	x=$$(find . -name \*.py -not -empty -type f -print0 | xargs -0 grep -L "^# Copyright"); \
```

Comment:
ah, `-L`, good one

Reply:
Indeed, this can almost just be "grep -r" but wanting to skip empty files complicated it a bit.